Orvibo.js

```diff
@@ -2,23 +2,22 @@ const net = require('net');
 const Packet = require('./OrviboPacket');
 const PacketBuilder = require('./PacketBuilder');
 const Utils = require('./Utils');
-const Settings = require('./OrviboSettings');
```

Comment:
Really like the idea of the config.json but we might need to still support the OrviboSettings file for people who might upgrade the library. Or maybe we can mark it as a breaking change. What do you think?